jina/hubble/hubio.py

```diff
@@ -415,16 +415,16 @@ def _async_push(
         new_task_id = push_task.get('_id')
         if new_task_id:
 
-            dump_secret(work_path, None, None, new_task_id)
+            dump_secret(work_path, None, form_data.get('secret'), new_task_id)
```

Comment:
This drops uuid8

Reply:
mmm, we use task_id to keep track of the id so the id is not lost

Reply:
But it prevents the user from updating the executor before the last update is complete.
However, it could never make it to completion for some reason.

Reply:
emm, I see, and  I will add  convert `dump_secret(work_path, None, form_data.get('secret'), new_task_id)` to `dump_secret(work_path,  form_data.get('id'), form_data.get('secret'), new_task_id)`, then it will keep the id and will not depend on the last commit  is complete.